### PR TITLE
Release v3.3.0

### PR DESCRIPTION
**Pull request by tosbaha**

### CODE REVIEW

File: docs/updates/release-notes.md [hunk 1]
```diff
@@ -11,6 +11,23 @@ import CloudBadge from '@site/src/components/CloudBadge';
...
+- [Slack Bot](../account/slack/appcircle-slack-bot.md) added. <CloudBadge/> <SelfHostedBadge/>
+
+### :muscle: Improvement
+- SVG images are updated <CloudBadge/> <SelfHostedBadge/>
+- Build profile card design is improved. <CloudBadge/> <SelfHostedBadge/>
+- Color scheme and icons are updated for dark themes. <CloudBadge/> <SelfHostedBadge/>
+
+### 🐞 Fixed
+- [Huawei AppGallery](../store-submit/huawei-app-gallery.md) submission bug fixed.  <SelfHostedBadge/>
+- [Enterprise Store](../enterprise-appstore/customize-ent-store.md) 2FA localization bug fixed. <SelfHostedBadge/>
```
Comment (csonuryilmaz) [inline docs/updates/release-notes.md:29]: @tosbaha I could not find any issue for this item in April roadmap.

We can replace it with below note. (BE-1256)
```suggestion
- [Enterprise Store](../enterprise-appstore/customize-ent-store.md) The background image bug was fixed on the login page. <SelfHostedBadge/>
```

File: docs/updates/release-notes.md [hunk 2]
```diff
@@ -11,6 +11,23 @@ import CloudBadge from '@site/src/components/CloudBadge';
...
+- Latest five build status added to build profile. <CloudBadge/> <SelfHostedBadge/>
+- [Slack Bot](../account/slack/appcircle-slack-bot.md) added. <CloudBadge/> <SelfHostedBadge/>
+
+### :muscle: Improvement
+- SVG images are updated <CloudBadge/> <SelfHostedBadge/>
+- Build profile card design is improved. <CloudBadge/> <SelfHostedBadge/>
+- Color scheme and icons are updated for dark themes. <CloudBadge/> <SelfHostedBadge/>
+
+### 🐞 Fixed
+- [Huawei AppGallery](../store-submit/huawei-app-gallery.md) submission bug fixed.  <SelfHostedBadge/>
```
Comment (csonuryilmaz) [inline docs/updates/release-notes.md:28]: :warning: We need to wait for the discussion to be resolved for this item. See details in [here](https://appcircleteam.slack.com/archives/C02RXLSMU3F/p1682693627798339).

File: docs/updates/release-notes.md [hunk 3]
```diff
@@ -11,6 +11,23 @@ import CloudBadge from '@site/src/components/CloudBadge';
...
+### 🆕 New Feature
+- [Data Theorem Mobile Secure](../workflows/common-workflow-steps.md#data-theorem-mobile-secure) workflow step added. <CloudBadge/> <SelfHostedBadge/>
+- [App Center CodePush](../workflows/react-native-specific-workflow-steps.md#app-center-codepush) workflow step added. <CloudBadge/> <SelfHostedBadge/>
+- Latest five build status added to build profile. <CloudBadge/> <SelfHostedBadge/>
+- [Slack Bot](../account/slack/appcircle-slack-bot.md) added. <CloudBadge/> <SelfHostedBadge/>
+
+### :muscle: Improvement
+- SVG images are updated <CloudBadge/> <SelfHostedBadge/>
+- Build profile card design is improved. <CloudBadge/> <SelfHostedBadge/>
+- Color scheme and icons are updated for dark themes. <CloudBadge/> <SelfHostedBadge/>
```
Comment (csonuryilmaz) [inline docs/updates/release-notes.md:25]: To mention general UI improvements done in April.

```suggestion
- Color scheme and icons are updated for dark themes. <CloudBadge/> <SelfHostedBadge/>
- Lots of UI and text improvements were made for better UX. <CloudBadge/> <SelfHostedBadge/>
```

File: docs/updates/release-notes.md [hunk 4]
```diff
@@ -11,6 +11,23 @@ import CloudBadge from '@site/src/components/CloudBadge';
 
 # Latest Release Notes
 
+## 3.3.0 - 2023-04-27 - Data Theorem Mobile Secure, App Center CodePush
+
+### 🆕 New Feature
+- [Data Theorem Mobile Secure](../workflows/common-workflow-steps.md#data-theorem-mobile-secure) workflow step added. <CloudBadge/> <SelfHostedBadge/>
+- [App Center CodePush](../workflows/react-native-specific-workflow-steps.md#app-center-codepush) workflow step added. <CloudBadge/> <SelfHostedBadge/>
+- Latest five build status added to build profile. <CloudBadge/> <SelfHostedBadge/>
+- [Slack Bot](../account/slack/appcircle-slack-bot.md) added. <CloudBadge/> <SelfHostedBadge/>
```
Comment (csonuryilmaz) [inline docs/updates/release-notes.md:20]: Currently slack bot is not supported on self-hosted. It's in future roadmap.
```suggestion
- [Slack Bot](../account/slack/appcircle-slack-bot.md) added. <CloudBadge/>
```

**Review by csonuryilmaz — CHANGES_REQUESTED** [review]
I reviewed releases done in April, including hotfixes. I added some notes that I noticed.

File: docs/updates/release-notes.md (hunk 2 above)
Comment (tosbaha) [reply docs/updates/release-notes.md:29]: It's fixed today.

**Pull request merged**